lib/web/scripts/node-join/install.sh

```diff
@@ -15,9 +16,11 @@ MACOS_STDERR_LOG="/var/log/teleport-stderr.log"
 MACOS_STDOUT_LOG="/var/log/teleport-stdout.log"
 SYSTEMD_UNIT_PATH="/lib/systemd/system/teleport.service"
 TARGET_PORT_DEFAULT=443
+TELEPORT_APP_BUNDLES_darwin="tctl.app tsh.app"
 TELEPORT_ARCHIVE_PATH='{{.packageName}}'
 TELEPORT_BINARY_DIR="/usr/local/bin"
 TELEPORT_BINARY_LIST="teleport tctl tsh teleport-update"
+TELEPORT_BINARY_LIST_darwin="teleport"
```

Comment:
Not critical because there's only one item, but this should be declared as an array if it's going to be used as one below.